datadog_checks_dev/datadog_checks/dev/tooling/commands/validate/manifest.py

```diff
@@ -3,279 +3,29 @@
 # Licensed under a 3-clause BSD style license (see LICENSE)
 import json
 import os
-import uuid
 
 import click
-import jsonschema
+
+from datadog_checks.dev.tooling.manifest_validator.validator import get_all_validators
 
 from ....fs import file_exists, read_file, write_file
 from ...constants import get_root
-from ...git import content_changed
-from ...utils import get_metadata_file, parse_version_parts, read_metadata_rows
 from ..console import CONTEXT_SETTINGS, abort, echo_failure, echo_info, echo_success, echo_warning
 
-FIELDS_NOT_ALLOWED_TO_CHANGE = ["integration_id", "display_name", "guid"]
-
-METRIC_TO_CHECK_WHITELIST = {
-    'openstack.controller',  # "Artificial" metric, shouldn't be listed in metadata file.
-    'riakcs.bucket_list_pool.workers',  # RiakCS 2.1 metric, but metadata.csv lists RiakCS 2.0 metrics only.
-}
-
-
-def get_manifest_schema():
```

Comment:
Moved to schema.py https://github.com/DataDog/integrations-core/pull/9111/files#diff-b027c329eca59e9e002b48889d1a1658bf0746e27e1d2eb3921b682f5ed5c0dfR7